openmc/material.py

```diff
@@ -332,13 +332,13 @@ def get_all_nuclides(self):
 
         return nuclides
 
-    def _repr__(self):
+    def __repr__(self):
         string = 'Material\n'
         string += '{0: <16}{1}{2}\n'.format('\tID', '=\t', self._id)
         string += '{0: <16}{1}{2}\n'.format('\tName', '=\t', self._name)
 
         string += '{0: <16}{1}{2}'.format('\tDensity', '=\t', self._density)
-        string += ' [{0}]\n'.format(self._density_units)
+        string += ' ["{0}"]\n'.format(self._density_units)
```

Comment:
Definitely agree, removed them. The search-replace macros I used for this got a little out of hand in a few places evidently :-)